lib/rules/no-tabs.js

```diff
@@ -21,25 +21,45 @@ module.exports = {
             category: "Stylistic Issues",
             recommended: false
         },
-        schema: []
+        schema: [],
+        fixable: "whitespace"
     },
 
     create(context) {
         return {
             Program(node) {
+                let totalLength = 0;
+
                 context.getSourceLines().forEach((line, index) => {
-                    const match = regex.exec(line);
+                    let match;
+
+                    // Intentional use of conditional assignment - .exec() returns null when it
+                    // cannot find more matches, breaking the loop
+                    // eslint-disable-next-line no-cond-assign
+                    while (match = regex.exec(line)) {
```

Comment:
The `eslint-disable-next-line` might be removable if you surround the assignment in a (redundant) set of parentheses. That's a convention indicating the assignment is intentional. I know the rule *supports* this convention, though I don't know for sure if we have it configured that way. Something to look at, anyway.